data/migrations/20181116120932_projects.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex, Promise) {
```

Comment:
Migrations table looks great.  Also, defaulting to 0 is the same as defaulting to false.